bin/envsetup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Minor comment but having a bin folder in go project is less standard as in Rails. Since there is already a Makefile, how about moving that command to  that file too? 🤔 

```
dev-start:
   docker-compose -f docker-compose.dev.yml up -d 

test:
	go test ./...
```

Reply:
Good catch. Fixed in 2e5a4248963b42ab8b16ea9be1322c2ee10a27b7